elide-spring/elide-spring-boot-autoconfigure/pom.xml

```diff
@@ -239,6 +239,19 @@
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-jar-plugin</artifactId>
             </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
```

Comment:
We don't want to pull this in the autoconfigure package.  Shading should only be done by the end application.